scripts/build.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+function cleanup {
+  rm -rf build
+  rm AppleDevIDApp.p12
+  rm notarize_config.json
+  security delete-keychain percy.keychain
+}
+
+npm install -g pkg
+echo "ENVIRONMENT='executable'" > .env
+
+yarn build
+
+yarn install
+
+# Remove type from package.json files
+sed -i '' '/"type": "module",/d' ./package.json
+cd packages && sed -i '' '/"type": "module",/d' ./*/package.json && cd ..
+
+echo "import { cli } from '@percy/cli';\
+import 'dotenv/config';\
+$(cat ./packages/cli/dist/percy.js)" > ./packages/cli/dist/percy.js
+
+# Convert ES6 code to cjs
+npm run build_cjs
+cp -R ./build/* packages/
+
+# Create executables
+pkg ./packages/cli/bin/run.js -d
+
+# Rename executables
+mv run-macos percy-macos
+mv run-linux percy-linux
+mv run-win.exe percy-win.exe
+
+# Sign & Notrize mac app
+echo "$APPLE_DEV_CERT" | base64 -d > AppleDevIDApp.p12
+
+security create-keychain -p percy percy.keychain
```

Comment:
Add in confluence, link existing if already present.

Reply:
Added in https://browserstack.atlassian.net/wiki/spaces/ENG/pages/3818786920/Percy+SDK+Engineering+Handbook